soapysdr-sys/bindings/bindings_linux.rs

```diff
@@ -0,0 +1,131 @@
+// linux tests + data structures
+
+#[doc = "! Possible data types for argument info"]
+pub type SoapySDRArgInfoType = ::std::os::raw::c_uint;
+
+#[doc = " The available priority levels for log messages.\n\n The default log level threshold is SOAPY_SDR_INFO.\n Log messages with lower priorities are dropped.\n\n The default threshold can be set via the\n SOAPY_SDR_LOG_LEVEL environment variable.\n Set SOAPY_SDR_LOG_LEVEL to the string value:\n \"WARNING\", \"ERROR\", \"DEBUG\", etc...\n or set it to the equivalent integer value."]
+pub type SoapySDRLogLevel = ::std::os::raw::c_uint;
+
```

Comment:
These are the same on all platforms?

Reply:
On windows they are signed, oddly (`c_int` vs `c_uint`). If they're the same width, then it seems it would generally be the same with a potential footgun that might eventually be triggered. I'm not sure it's worth preserving this discrepancy but I figured I'd at least surface it in this PR.